app/resources/views/pages/location_report/collapsed.blade.php

```diff
@@ -22,8 +22,9 @@
 		'location' => $location))
 	<div class="basic-info">
 		<div class="addresses">
-			<address>{{ $location->address }}</address>
-			<a href="{{ $location->getExternalWebURL() }}">{{ $location->getExternalWebURL() }}</a>
+		<p><em class="fa fa-map-marker"></em>{{ $location->address }}</p>
+		@if(!is_null($location->phone_number))<p><em class="fa fa-phone"></em>{{ $location->phone_number }}</p>@endif
```

Comment:
Despite the name "address", the address element is for any kind of contact information.   For this reason, I suggest putting the phone number link inside the <address> element.

Official documentation on "address" elements are at:
https://www.w3.org/TR/2012/WD-html-markup-20120320/address.html